src/apollo/offline-queue-link/index.ts

```diff
@@ -155,31 +160,49 @@ export class OfflineQueueLink extends ApolloLink {
 		});
 	}
 
-	/** retry queries made while offline like apollo-link-queue */
-	retry = () => {
-		this.operationQueue.forEach(entry => {
-			const { operation, forward, observer } = entry;
-
-			// Wrap the observer to call dequeue on error/complete
-			forward(operation).subscribe({
-				...observer,
-				error: (err: any) => {
-					this.dequeue(entry);
-					if (observer.error) {
-						console.error(
-							'[OfflineQueueLink] Could not sync operation to server:',
-							operation
-						);
-						observer.error(err);
-					}
-				},
-				complete: () => {
-					this.dequeue(entry);
-					if (observer.complete) {
-						observer.complete();
-					}
+	// Retry queries made while offline like apollo-link-queue
```

Comment:
("Actually, everybody likes a pedant" voice) Can you add a period to the end of this line?